packages/cycle-scripts-one-fits-all/template/src/drivers.ts

```diff
@@ -0,0 +1,59 @@
+import xs, { Stream } from 'xstream';
+import { restartable } from 'cycle-restart';
+import { makeDOMDriver, VNode, DOMSource } from '@cycle/dom';
+import { makeHTTPDriver, HTTPSource, RequestOptions } from '@cycle/http';
+import { timeDriver, TimeSource } from '@cycle/time';
+import { makeRouterDriver, RouterSource, RouteMatcher } from 'cyclic-router';
+import { createBrowserHistory } from 'history';
+import switchPath from 'switch-path';
+//import storageDriver from '@cycle/storage'
```

Comment:
aha - I was wondering about that a I recalled yo usaid it was injected into each page. Thought don't .d.ts pages have special format / contents? Will look at that. Thanks